eng/Version.Details.xml

```diff
@@ -378,9 +378,9 @@
       <Sha>1ed3c1b184f9da5b9f71097672b63edb00858fad</Sha>
       <SourceBuild RepoName="roslyn" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.ApiCompat.Task" Version="10.0.100-alpha.1.24480.5">
+    <Dependency Name="Microsoft.DotNet.ApiCompat.Task" Version="10.0.100-alpha.1.24510.13">
       <Uri>https://github.com/dotnet/sdk</Uri>
-      <Sha>9de571282093d76401a3f457a8e9f4427e088778</Sha>
+      <Sha>1f220e48e5bc666e992b24f118a17e01ca316819</Sha>
     </Dependency>
     <!-- Intermediate is necessary for source build. -->
     <Dependency Name="Microsoft.SourceBuild.Intermediate.sdk" Version="10.0.100-alpha.1.24480.5">
```

Comment:
This package version also needs to get updated. It needs to be in sync with and other dotnet/sdk dependencies in this file.

Reply:
Thanks.

Reply:
Do I need to manually update the Sha too?

Reply:
Looks like the ApiCompat.Task version and sha upgrade are the ones that need to be downgraded as for whatever reason, there is not a newer `Microsoft.SourceBuild.Intermediate.sdk` package.